README.md

```diff
@@ -54,13 +54,15 @@ yarn add @auth0/auth0-spa-js
 //with async/await
 const auth0 = await createAuth0Client({
   domain: '<AUTH0_DOMAIN>',
-  client_id: '<AUTH0_CLIENT_ID>'
+  client_id: '<AUTH0_CLIENT_ID>',
+  redirect_uri: 'https://myapp.com/callback'
```

Comment:
Should probably follow the same convention as the other parameters, i.e.  `<MY_CALLBACK_URL>`

Reply:
done